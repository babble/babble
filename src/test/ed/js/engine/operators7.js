a = 5
print( ( a += 1 ) )
print( a );

a = { a : 1 };
a.a += 1;

print( ( a.a += 1 ) );
print( a.a );


a = 5
print( ( a -= 1 ) )
print( a );

a = { a : 3 };
a.a -= 1;
print( ( a.a -= 1 ) );
print( a.a );



a = 5
print( ( a *= 1 ) )
print( a );

a = { a : 3 };
a.a *= 1;
print( ( a.a *= 1 ) );
print( a.a );



a = 5
print( ( a /= 2 ) )
print( a );

a = { a : 3 };
a.a /= 2;
print( ( a.a /= 2 ) );
print( a.a );


