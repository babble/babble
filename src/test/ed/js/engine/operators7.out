6
6
3
3
4
4
1
1
5
5
3
3
2.5
2.5
0.75
0.75
